tests/Tests_RangeEnhancedGenerator.py

```diff
@@ -0,0 +1,48 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+
+import unittest
+import RangeEnhancedIterator_Generator
+from RangeEnhancedIterator_Generator import GeneratorEnhancedRange
+
+
+class GeneratorTests(unittest.TestCase):
+
+    def setUp(self):
+        print ('Setup called')
```

Comment:
Do we need this message?

Reply:
Seems that we don't. I was just in Alexander's template.